zetaclient/chains/interfaces/interfaces.go

```diff
@@ -39,15 +39,21 @@ const (
 
 // ChainObserver is the interface for chain observer
 type ChainObserver interface {
+	// Start starts the observer
 	Start(ctx context.Context)
+
+	// Stop stops the observer
 	Stop()
-	VoteOutboundIfConfirmed(
-		ctx context.Context,
-		cctx *crosschaintypes.CrossChainTx,
-	) (bool, error)
+
+	// ChainParams returns observer chain params (might be out of date with zetacore)
+	ChainParams() observertypes.ChainParams
+
+	// SetChainParams sets observer chain params
 	SetChainParams(observertypes.ChainParams)
-	GetChainParams() observertypes.ChainParams
-	WatchInboundTracker(ctx context.Context) error
```

Comment:
Shall we keep this method. Inbound tracker ticker is still necessary ATM.

Reply:
> Shall we keep this method?

No, because it's a concern of a specific observer and should be incapsulated within `observer.Start()`.

Reply:
got it